bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AZHToLLtt/AZHToLLtt_customizecards.dat

```diff
@@ -0,0 +1,11 @@
+set param_card frblock 1 1.0 #tanbeta 
+set param_card frblock 2 0.99  #sinbma
+set param_card mass 25 125.0
+set param_card mass 35 HMASS
+set param_card mass 36 AMASS
+set param_card mass 37 AMASS
+set param_card mass 23 9.118760e+01
+set param_card mass 24 80.379
+set param_card width 35 Auto
+set param_card width 23 2.4952
+set param_card width 37 Auto
```

Comment:
since w/z properties here are the same with those in the default model parameter card, line 7/8/10 could be removed. "set param_card width 36 Auto" should be added.